Frameworks/UIKit/UIScrollView.mm

```diff
@@ -169,87 +162,29 @@ - (id)_dManipGestureCallback:(_UIDMPanGestureRecognizer*)gesture {
 
 - (void)_initUIScrollView {
     // Store a strongly-typed backing scrollviewer
-    _scrollViewer = rt_dynamic_cast<WXCScrollViewer>([self xamlElement]);
-    if (!_scrollViewer) {
+    WXCGrid* scrollViewGrid = rt_dynamic_cast<WXCGrid>([self xamlElement]);
+    WXCScrollViewer* scrollViewer = nil;
```

Comment:
_scrollViewer is initialized using  scrollViewer.  

---
In reply to: [106549343](https://github.com/Microsoft/WinObjC/pull/2257#discussion_r106549343) [](ancestors = 106549343)